content/prerequisites/local-development/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Local Development Setup"
+weight: 300
+---
+
```

Comment:
It might be good to point out this is not mandatory if using Cloud9

```suggestion

:::alert{type="info"}
This is only required if you are not using Cloud9 IDE for the workshop.
:::

```